### DISCUSSION TITLE
Multiple devices same IP address

### DISCUSSION THREAD

**Discussion by AndreasMoe** [netbox-community/netbox#11719] [Q&A]
Hi,
We have multi site configuration and we sometime have same IP on different devices. Is there anyway to do this ? Using scripts to assign IPs to VMs and Devices. If multiple devices have same IP, how do I choose which IP to use if multiple is found?

Also tried to create two prefix'es, one for each site, but if added same IP, the IP is shown as used in both prefixes

Tried to search google about this, but just got confused :)

**Comment by jsenecal** [comment]
You would need to have the same prefix defined in multiple different VRF, one per site.
